src/utilities/getIntrospectionQuery.ts

```diff
@@ -32,6 +32,10 @@ export interface IntrospectionOptions {
    * Default: false
    */
   inputValueDeprecation?: boolean;
+
+  // Whether target GraphQL server supports `@oneOf` input objects.
+  // Default: false
```

Comment:
should this be JSDoc formatted to match the comments above?

Reply:
🤦🏻 definitely. updated in [3af4434](https://github.com/graphql/graphql-js/pull/4078/commits/3af443467cdb9713f700e2b7b4368fb55c18171c)